### DISCUSSION TITLE
Add content scripts/css

### DISCUSSION THREAD

**Discussion by mmkal** [getgauge/taiko#2603] [General]
We'd like to test a private chrome extension that we've built. All it does is add a content script and a css file to certain webpages. So we could bypass the actual extension by just adding `<link>` and `<script>` tags directly to the `<head>` element. What's the best way to do this with Taiko?

**Comment by zabil** [comment]
Chrome has a `--load-extension` option https://peter.sh/experiments/chromium-command-line-switches/. You can try passing this option to [`openBrowser`](https://docs.taiko.dev/api/openbrowser/)

**Reply by mmkal** [reply]
Will try this. Thanks!

**Reply by mmkal** [reply]
@zabil this works really nicely, thank you! One snag I've found, though - I can't find a way to make the extension work in incognito mode. I looked for a switch on the page you linked that's the equivalent of the "Allow in Incognito" UI on the chrome://extensions page, but I couldn't find it:

<img width="749" alt="image" src="https://user-images.githubusercontent.com/15040698/177777456-0a33de11-8b69-4c37-8243-6ec11cf049fe.png">
